### PR TITLE
feat: cookie banner

### PR DESCRIPTION
**Pull request by Flosciante**

### CODE REVIEW

File: components/theme/app/AppLayout.vue [hunk 1]
```diff
@@ -15,8 +16,9 @@
         <slot />
       </div>
     </div>
+    <AppFooter :links="footerLinks" :class="showCookieBanner ? 'pb-16 md:pb-12' : ''" />
 
-    <AppFooter :links="footerLinks" />
+    <CookieBanner class="w-full fixed bottom-0 left-0 z-40" @cookie-banner="showCookieBanner = false" />
```
Comment (smarroufin) [inline components/theme/app/AppLayout.vue:21]: Really minor comment, but I feel like doing `inset-x-0` is more secure and simpler than `w-full left-0` 🙂

File: components/templates/CookieBanner.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          </SectionButton>
+        </template>
+      </CookieConsent>
+    </VueIfBot>
+  </div>
+</template>
+
+<script>
+import VueIfBot from 'vue-if-bot/dist/vue-if-bot.es'
+import CookieConsent from 'vue-cookieconsent-component/src/components/CookieConsent.vue'
```
Comment (smarroufin) [inline components/templates/CookieBanner.vue:54]: According to the docs it does not seem right to import them this way. Is there a reason?
https://github.com/Developmint/vue-if-bot#synchronous-import
https://github.com/EvodiaAut/vue-cookieconsent-component#usage

**Review by smarroufin — CHANGES_REQUESTED** [review]
Out of thoses two comments, it seems fine and works well. Probably wait for a second review, from someone actually knowing cookies 😅 .

File: components/theme/app/AppLayout.vue (hunk 1 above)
Comment (Flosciante) [reply components/theme/app/AppLayout.vue:21]: 👍

File: components/templates/CookieBanner.vue (hunk 2 above)
Comment (manniL) [reply components/templates/CookieBanner.vue:54]: I think I have to update the docs 🙈 
`vue-if-bot` import seems alright 👍 
https://github.com/manniL/lichter.io/blob/31cded247d5c2cad2bb6f29be5355e46bec9ffdb/layouts/default.vue#L18

**Pull request merged**